website/content/docs/connect/observability/grafanadashboards/index.mdx

```diff
@@ -0,0 +1,115 @@
+---
+layout: docs
+page_title: Service Mesh Observability - Dashboards
+description: >-
+  This documentation provides an overview of several dashboards designed for monitoring and managing services within a Consul-managed Envoy service mesh. Learn how to enable access logs and configure key performance and operational metrics to ensure the reliability and performance of services in the service mesh.
+---
+
+# Dashboards for service mesh observability
+
+This topic describes the configuration and usage of dashboards for monitoring and managing services within a Consul-managed Envoy service mesh. These dashboards provide critical insights into the health, performance, and resource utilization of services. The dashboards described here are essential tools for ensuring the stability, efficiency, and reliability of your service mesh environment.
+
+## Dashboards Overview
+
+The repository includes the following dashboards:
+
+  - **Consul Service-to-Service Dashboard**: Provides a detailed view of service-to-service communications, monitoring key metrics like access logs, HTTP requests, error counts, response code distributions, and request success rates. The dashboard includes customizable filters for focusing on specific services and namespaces.
+  
+  - **Consul Service Dashboard**: Tracks key metrics for Envoy proxies at the cluster and service levels, ensuring the performance and reliability of individual services within the mesh.
+
+  - **Consul DataPlane Dashboard**: Offers a comprehensive overview of service health and performance, including request success rates, resource utilization (CPU and memory), active connections, and cluster health. It helps operators maintain service reliability and optimize resource usage.
+    
+  - **Consul K8s Dashboard**: Focuses on monitoring the health and resource usage of the Consul control plane within a Kubernetes environment, ensuring the stability of the control plane.
+
+  - **Consul Server Dashboard**: Provides detailed monitoring of Consul servers, tracking key metrics like server health, CPU and memory usage, disk I/O, and network performance. This dashboard is critical for ensuring the stability and performance of Consul servers within the service mesh.
+
+## Enabling Observability
```

Comment:
```suggestion
## Enabling Observability
The Helm chart provides configuration to enable a demo Prometheus server. https://developer.hashicorp.com/consul/docs/k8s/helm#prometheus
```

Reply:
@boruszak I think the above may be all that we can say here? Basically we *can* install a demo prometheus server, but it is really on the user to deploy Prometheus/Loki/Grafana, and just upload our dashboards into Grafana.